stdlib/platform/java/thread/ThreadHelper.wyv

```diff
@@ -0,0 +1,27 @@
+module def ThreadHelper(java : Java) 
```

Comment:
Wyvern convention is that modules (as opposed to types) should start lowercase, because they are values.